src/test/java/pages/CheckoutPage.java

```diff
@@ -7,6 +7,7 @@
 import static com.codeborne.selenide.Selenide.$$x;
 import static com.codeborne.selenide.Selenide.$x;
 
+
 public class CheckoutPage extends BasePage{
     private static SelenideElement firstNameInput = $x("//input[@id='BillingNewAddress_FirstName']");
     private static SelenideElement lastNameInput = $x("//input[@id='BillingNewAddress_LastName']");
```

Comment:
```suggestion
    private static String locatorPattern = "//input[@id='BillingNewAddress_%s']"
    private static SelenideElement lastNameInput = $x(String.format( locatorPattern, "LastName" ));
```
выносим общую часть локаторов в отдельную строку с использованием форматирования